### PR TITLE
consistent between docs and demo code

### PR DESCRIPTION
**Pull request by zhouhang95**

### CODE REVIEW

**Comment by jin-s13** [comment]
Thanks!
Maybe `out-video-root` is a better name.

File: docs/getting_started.md [hunk 1]
```diff
@@ -176,7 +176,7 @@ python demo/top_down_video_demo_with_mmdet.py mmdetection/configs/faster_rcnn/fa
     mmdetection/checkpoints/faster_rcnn_r50_fpn_1x_coco_20200130-047c8118.pth \
     configs/top_down/hrnet/coco/hrnet_w48_coco_256x192.py \
     hrnet_w48_coco_256x192-b9e0b3ab_20200708.pth \
-    --video_path demo/demo_video.mp4 \
+    --video-path demo/demo_video.mp4 \
     --output-video-root vis_results
```
Comment (jin-s13) [inline docs/getting_started.md:180]: Better to change this `output-video-root` to `out-video-root`

File: demo/bottom_up_video_demo.py [hunk 2]
```diff
@@ -19,7 +19,7 @@ def main():
         default=False,
         help='whether to show visualizations.')
     parser.add_argument(
-        '--out-video-root',
+        '--output-video-root',
```
Comment (jin-s13) [inline demo/bottom_up_video_demo.py:22]: It is better to keep this name as it is, and change the docs.

**Review by jin-s13 — APPROVED** [review]

**Comment by innerlee** [comment]
We can make the scripts more consistent by imposing this `abc-def-gh`  convention on all scripts in tools folder. like here https://github.com/open-mmlab/mmpose/blob/master/tools/test.py#L28

it can be done in a separate pr

**Pull request merged**